features/package-install.feature

```diff
@@ -1,5 +1,35 @@
 Feature: Install WP-CLI packages
 
+  Scenario: Install a package with an old package index url in package composer.json
+    Given an empty directory
+    And a composer.json file:
```

Comment:
> Although, there's really no need to put it into a sub-directory, we can just treat the "empty" directory as the package-directory itself.

This seems like it may cause problems.